src/components/EditCoursePage/__snapshots__/CollapsibleCourseRun.test.jsx.snap

```diff
@@ -67,15 +67,15 @@ exports[`Collapsible Course Run renders correctly variant_id field for external
                 className="text-primary ml-2"
                 fade={false}
                 fixedWidth={false}
-                flip={null}
+                flip={false}
                 icon={
                   Object {
                     "icon": Array [
-                      512,
+                      448,
                       512,
                       Array [],
                       "f0c5",
-                      "M502.6 70.63l-61.25-61.25C435.4 3.371 427.2 0 418.7 0H255.1c-35.35 0-64 28.66-64 64l.0195 256C192 355.4 220.7 384 256 384h192c35.2 0 64-28.8 64-64V93.25C512 84.77 508.6 76.63 502.6 70.63zM464 320c0 8.836-7.164 16-16 16H255.1c-8.838 0-16-7.164-16-16L239.1 64.13c0-8.836 7.164-16 16-16h128L384 96c0 17.67 14.33 32 32 32h47.1V320zM272 448c0 8.836-7.164 16-16 16H63.1c-8.838 0-16-7.164-16-16L47.98 192.1c0-8.836 7.164-16 16-16H160V128H63.99c-35.35 0-64 28.65-64 64l.0098 256C.002 483.3 28.66 512 64 512h192c35.2 0 64-28.8 64-64v-32h-47.1L272 448z",
+                      "M384 336H192c-8.8 0-16-7.2-16-16V64c0-8.8 7.2-16 16-16l140.1 0L400 115.9V320c0 8.8-7.2 16-16 16zM192 384H384c35.3 0 64-28.7 64-64V115.9c0-12.7-5.1-24.9-14.1-33.9L366.1 14.1c-9-9-21.2-14.1-33.9-14.1H192c-35.3 0-64 28.7-64 64V320c0 35.3 28.7 64 64 64zM64 128c-35.3 0-64 28.7-64 64V448c0 35.3 28.7 64 64 64H256c35.3 0 64-28.7 64-64V416H272v32c0 8.8-7.2 16-16 16H64c-8.8 0-16-7.2-16-16V192c0-8.8 7.2-16 16-16H96V128H64z",
```

Comment:
why is snapshot update needed? In a previous change https://github.com/openedx/frontend-app-publisher/pull/869, this was not needed.

Reply:
i think this is because of the icon dependency on external library, it is recently updated by renovate,
https://github.com/openedx/frontend-app-publisher/blob/master/package-lock.json#L18